packages/ssr-web/app/components/common/degraded-service-banner/index.css

```diff
@@ -4,20 +4,17 @@
   --icon-background-color: var(--degraded-service-banner-text-color);
   --icon-color: var(--degraded-service-banner-background-color);
 
-  width: 100%;
   min-height: 2rem;
-  display: flex;
-  justify-content: center;
-  align-items: center;
```

Comment:
The removal of this makes it look a bit too high for me; here’s me twiddling this on and off:

![screencast 2022-03-25 10-17-35](https://user-images.githubusercontent.com/43280/160150115-4bc5b89a-df6d-4393-8878-e719a6483a36.gif)

Even with it on it looks maybe 1px too high for me but it’s less irksome at least 😆

Reply:
oh ACTUALLY I was setting this on `…_container`

Reply:
Ya vertical centering issues become more obvious with long text, thanks